engine/execution/ingestion/stop/stop_control_test.go

```diff
@@ -879,3 +879,8 @@ func Test_StopControlWorkers(t *testing.T) {
 		unittest.AssertClosesBefore(t, sc.Done(), 10*time.Second)
 	})
 }
+
+func TestPatchedVersion(t *testing.T) {
+	require.True(t, semver.New("0.31.20").LessThan(*semver.New("0.31.21")))
+	require.True(t, semver.New("0.31.20-patch.1").LessThan(*semver.New("0.31.20"))) // be careful with this one
+}
```

Comment:
Not sure we need this test. It could be left in as a warning, but it doesn't actually test any of our code.

Reply:
Yeah, it's not our code, but we need this test to confirm and understand its behavior.